pom.xml

```diff
@@ -24,8 +24,8 @@
   </licenses>
 
   <properties>
-    <jenkins.version>1.625</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.73.1</jenkins.version>
```

Comment:
Maybe better to use 2.60.3 as the oldest Java 8 LTS unless you need something specific from the 2.73 line.

Reply:
That was my first option, but the thing is that the plugins was compiled for that version by the CI since December, and I found that it is not the same to compile for 2.60.3 or for 2.73.1, I learn it on the hard way in this PR.

https://github.com/jenkinsci/ssh-slaves-plugin/commit/77ed6569ea7e72a87029709bc288efcbe8a8c7a5#diff-58231b16fdee45a03a4ee3cf94a9f2c3

Reply:
FWIW `jenkinsVersions: [null, '2.73.1']` will build the plugin using the baseline in the pom and 2.73.1, so it should have been building on 1.625 and 2.73.1. I'm surprised there would be a big difference between 2.60 and 2.73 though.